index.html

```diff
@@ -23,10 +23,7 @@
 	  github: "w3c/mathml-aam",
 	  copyrightStart:  "2020",
 
-	  wg: "Accessible Rich Internet Applications Working Group",
-	  wgURI: "https://www.w3.org/WAI/ARIA/",
-	  wgPublicList: "public-aria",
```

Comment:
wgPublicList is not deprecated: https://respec.org/docs/#wgPublicList

Is group: "aria" enough to deduce the public mailing list? It didn't seem to be the case when I tested.

Reply:
I prefer to direct feedback through github so don't like the links at the top. There are still links in the SOTD section for anyone that needs the mailing list. 
Editors decision if you want this I believe.